Sources/BraveWallet/Crypto/Stores/KeyringStore.swift

```diff
@@ -251,6 +261,7 @@ public class KeyringStore: ObservableObject {
   }
 
   func restoreWallet(phrase: String, password: String, isLegacyBraveWallet: Bool, completion: ((Bool) -> Void)? = nil) {
+    isRestoringWallet = true
```

Comment:
should this get reset once its completed? cc @StephenHeaps

Reply:
Commit got squashed so it’s hard to see the full change, but it resets in `markOn oardingAsCompleted()`
https://github.com/brave/brave-ios/blob/feature/MultiWindow/Sources/BraveWallet/Crypto/Stores/KeyringStore.swift#L193